FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheck.h

```diff
@@ -38,6 +51,20 @@ NS_SWIFT_NAME(AppCheck)
 /// @throw Throws an exception if required `FirebaseApp` options are missing.
 + (nullable instancetype)appCheckWithApp:(FIRApp *)firebaseApp NS_SWIFT_NAME(appCheck(app:));
 
+/// Requests Firebase app check token. This method should *only* be used if you need to authorize
+/// requests to a non-Firebase backend. Requests to Firebase backend are authorized automatically if
+/// configured.
+/// @param forcingRefresh If `YES`,  a new Firebase app check token is requested and the token
+/// cache is ignored. If `NO`, the cached token is used if it exists and has not expired yet. In
+/// most cases, `NO` should be used. `YES` should only be used if the server explicitly returns an
+/// error, indicating a revoked token.
+/// @param handler The completion handler. Includes the app check token if the request succeeds,
+/// or an error if the request fails.
+- (void)tokenForcingRefresh:(BOOL)forcingRefresh
+                 completion:
+                     (void (^)(FIRAppCheckToken *_Nullable token, NSError *_Nullable error))handler
+    NS_SWIFT_NAME(token(forcingRefresh:completion:));
```

Comment:
This API's signature aligns with the amended API proposal. 

Usage
```swift
// Completion handler
  let token = AppCheck.appCheck().token(forcingRefresh: true) { token, error in
    // ...
  }

// Async await
let token = try await AppCheck.appCheck().token(forcingRefresh: true)
```
```objc
  [[FIRAppCheck appCheck] tokenForcingRefresh:NO
                                   completion:^(FIRAppCheckToken *_Nullable token, NSError *_Nullable error) {
                                      // ...
                                   }];
```

Reply:
I've updated the source to reflect the amended API and notified the API team yesterday (there have been no updates to the thread). 

As a final check before merging, I just wanted to check I've taken required steps before merging a new API. 
cc: @ryanwilson @paulb777

Reply:
LGTM, it would be great to have a Swift API test.